virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc

```diff
@@ -5,68 +5,69 @@ include::modules/virt-document-attributes.adoc[]
 :virt-importing-vmware-vm:
 toc::[]
 
-You can import a single virtual machine or template from VMware vSphere 6.5, 6.7, or 7.0 into {VirtProductName}.
+You can import a VMware vSphere 6.5, 6.7, or 7.0 VM or VM template into {VirtProductName} by using the VM Import wizard.
 
-If you import a VMware template, {VirtProductName} creates a virtual machine based on the template.
+If you import a VM template, {VirtProductName} creates a virtual machine based on the template.
 
-The import process uses the VMware Virtual Disk Development Kit (VDDK) to copy the VMware virtual disk. You must download the VDDK SDK, build a VDDK image, upload image to your image registry, and add it to the `v2v-vmware` ConfigMap.
+include::modules/virt-features-for-storage-matrix.adoc[leveloffset=+1]
 
-include::modules/virt-importing-vm-prerequisites.adoc[leveloffset=+1]
+[id='preparing-a-vddk-image_{context}']
+== Preparing a VDDK image
 
-[discrete]
-include::modules/virt-features-for-storage-matrix.adoc[leveloffset=+2]
+The import process uses the VMware Virtual Disk Development Kit (VDDK) to copy the VMware virtual disk.
 
-== Configuring an image registry for the VDDK image
+You can download the VDDK SDK, create a VDDK image, upload the image to an image registry, and add it to the `v2v-vmware` ConfigMap.
 
-You can configure either an internal {product-title} image registry or a secure external image registry for the VDDK image.
+You can configure either an internal {product-title} image registry or a secure external image registry for the VDDK image. The registry must be accessible to your {VirtProductName} environment.
 
 [NOTE]
 ====
 Storing the VDDK image in a public registry might violate the terms of the VMware license.
 ====
 
+[id='configuring-an-internal-image-registry_{context}']
```

Comment:
Double quotes

There are a bunch in the PR that need to be fixed - I'm not going to point out all instances.